sdk/identity/identity/src/client/errors.ts

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+/**
+ * See the official documentation for more details:
+ *
+ * https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-protocols-oauth-code#error-response-1
```

Comment:
I wanted to put a link to the v2 documentation here but that page doesn't actually contain the HTTP status and error code information like the v1 page does.  Since the "error response" object shape looked the same I figured the error codes would likely be similar enough to be useful.  Happy to remove that link though if you think it will be confusing!